src/test/scala/com/feedzai/cosytest/junit/IntegrationSpec.scala

```diff
@@ -0,0 +1,55 @@
+package com.feedzai.cosytest.junit
+
+import java.nio.file.Paths
+
+import com.feedzai.cosytest.{DockerComposeRule, DockerComposeSetup}
+import org.junit.{BeforeClass, ClassRule, Test}
+import org.scalatest.MustMatchers
+import org.slf4j.LoggerFactory
+
+class IntegrationSpec extends MustMatchers {
```

Comment:
I think it’s better to have a JUnit test written in Java, since it’s more likely for JUnit to be used in Java.